src/Lecture1.hs

```diff
@@ -81,7 +85,11 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z = max (diff x y) (max (diff x z) (diff y z))
+    where
+        diff :: Int -> Int -> Int
+        diff a b = abs (abs a - abs b)
```

Comment:
That's an interesting solutions 🙂 

I think it would look a bit simpler if you find the max value among three arguments and the min value and find the difference between then. In that case, you won't need to use `abs` at all!

Reply:
Yes, makes sense 🤔🙂